### ISSUE TITLE
Low GPU utilization on NER model training 

### ISSUE THREAD

**Issue by selva221724** [flairNLP/flair#2703]
Hi, 

I am training a large dataset of `NER` model using `flair` and the GPU utilization is only 20- 30% when the training happens.

This is my model layer & settings

```
2022-04-04 14:20:21,426 ----------------------------------------------------------------------------------------------------
2022-04-04 14:20:21,427 Model: "SequenceTagger(
  (embeddings): StackedEmbeddings(
    (list_embedding_0): CharacterEmbeddings(
      (char_embedding): Embedding(275, 25)
      (char_rnn): LSTM(25, 25, bidirectional=True)
    )
    (list_embedding_1): BytePairEmbeddings(model=1-bpe-multi-100000-50)
  )
  (word_dropout): WordDropout(p=0.05)
  (locked_dropout): LockedDropout(p=0.5)
  (embedding2nn): Linear(in_features=650, out_features=650, bias=True)
  (rnn): LSTM(650, 256, batch_first=True, bidirectional=True)
  (linear): Linear(in_features=512, out_features=23, bias=True)
  (beta): 1.0
  (weights): None
  (weight_tensor) None
)"
2022-04-04 14:20:21,430 ----------------------------------------------------------------------------------------------------
2022-04-04 14:20:21,430 Corpus: "Corpus: 2948930 train + 2000 dev + 842571 test sentences"
2022-04-04 14:20:21,431 ----------------------------------------------------------------------------------------------------
2022-04-04 14:20:21,432 Parameters:
2022-04-04 14:20:21,433  - learning_rate: "0.1"
2022-04-04 14:20:21,434  - mini_batch_size: "32"
2022-04-04 14:20:21,434  - patience: "3"
2022-04-04 14:20:21,435  - anneal_factor: "0.5"
2022-04-04 14:20:21,436  - max_epochs: "5"
2022-04-04 14:20:21,436  - shuffle: "True"
2022-04-04 14:20:21,438  - train_with_dev: "False"
2022-04-04 14:20:21,440  - batch_growth_annealing: "False"
2022-04-04 14:20:21,440 ----------------------------------------------------------------------------------------------------
2022-04-04 14:20:21,441 Model training base path: "addressparser_data\100000\model"
2022-04-04 14:20:21,442 ----------------------------------------------------------------------------------------------------
2022-04-04 14:20:21,442 Device: cuda:0
2022-04-04 14:20:21,443 ----------------------------------------------------------------------------------------------------
2022-04-04 14:20:21,443 Embeddings storage mode: none
2022-04-04 14:20:21,446 ----------------------------------------------------------------------------------------------------
2022-04-04 15:04:22,217 epoch 1 - iter 9215/92155 - loss 0.12749600 - samples/sec: 111.71 - lr: 0.100000
2022-04-04 15:44:59,370 epoch 1 - iter 18430/92155 - loss 0.10006989 - samples/sec: 121.05 - lr: 0.100000
2022-04-04 16:30:37,388 epoch 1 - iter 27645/92155 - loss 0.09042490 - samples/sec: 107.74 - lr: 0.100000
2022-04-04 17:17:21,879 epoch 1 - iter 36860/92155 - loss 0.08745284 - samples/sec: 105.19 - lr: 0.100000
2022-04-04 17:59:28,028 epoch 1 - iter 46075/92155 - loss 0.08517210 - samples/sec: 116.78 - lr: 0.100000
2022-04-04 18:39:09,191 epoch 1 - iter 55290/92155 - loss 0.08944734 - samples/sec: 123.89 - lr: 0.100000
2022-04-04 19:23:13,211 epoch 1 - iter 64505/92155 - loss 0.09197077 - samples/sec: 111.57 - lr: 0.100000
```

I have `NVIDIA® TESLA® M60` GPU with 8GB RAM. 
cuda version: `11.4`

<img width="457" alt="image" src="https://user-images.githubusercontent.com/8901901/161622040-dab0105e-9a1c-419c-91ec-99a007dfc9f8.png">

Any suggestion to improve the GPU utilization and faster the training? 

<img width="200" alt="image" src="https://media4.giphy.com/media/3oz8xOu5Gw81qULRh6/giphy.gif">

**Comment by alanakbik** [comment]
Dear @selva221724 we just released flair 0.11 should be a bit faster if you use a CRF. Can you try? 

Aside from this, we still have CPU bottlenecks in our code that we are working on removing, but this will take some time.

**Comment by selva221724** [comment]
Thanks a lot @alanakbik . I will try that one.
